sources/web/datalab/polymer/components/datalab-settings/datalab-settings.html

```diff
@@ -14,21 +14,55 @@
 
 <link rel="import" href="../../components/shared-styles/shared-styles.html">
 
-<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/neon-animation/web-animations.html">
 <link rel="import" href="../../bower_components/paper-input/paper-input.html">
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/paper-tooltip/paper-tooltip.html">
+<link rel="import" href="../../bower_components/paper-progress/paper-progress.html">
 
 <dom-module id="datalab-settings">
   <template>
     <style include="datalab-shared-styles">
+      paper-progress {
+        width: 100%;
+        height: 1px;
+        --paper-progress-active-color: var(--selection-fg-color);
+        --paper-progress-container-color: var(--border-color);
+      }
       paper-radio-button {
         --paper-radio-button-checked-color: var(--selection-fg-color);
       }
       .setting {
         padding: 15px;
         border-bottom: 1px solid var(--border-color);
       }
+      .input-container {
+        display: flex;
+        height: 35px;
+      }
+      .input-container > * {
+        margin-right: 10px;
```

Comment:
Not really, this here is setting some spacing between the `input-container`'s elements, not for the `input-container` itself, and the elements are in row-layout.